manifests/init.pp

```diff
@@ -17,6 +19,20 @@
     purge   => $config_dir_purge,
   }
 
+  if str2bool($cron_purge) {
+    file {'/etc/cron.hourly/00awstats':
+      ensure => 'absent'
```

Comment:
I thought about that, but I think puppet might puke if I define a file as being absent and then also define the owner/group/mode/source.